rules/windows/process_creation/proc_creation_win_susp_schtasks_delete_all.yml

```diff
@@ -0,0 +1,25 @@
+title: Delete All Scheduled Tasks
+id: 220457c1-1c9f-4c2e-afe6-9598926222c1
+status: experimental
+description: Detects the usage of schtasks with the delete flag and the asterisk symbole to delete all tasks from the schedule of the local computer, including tasks scheduled by other users.
+author: Nasreddine Bencherchali
+references:
+    - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/schtasks-delete
+date: 2022/09/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\schtasks.exe'
+        CommandLine|contains|all:
+            - ' /delete '
+            - '/tn *'
```

Comment:
need the * or not ?
If yes  need to exclude -> \\*
Otherwise '/tn ' is enought

Reply:
It's needed. I updated the rule